sdk/communication/azure-communication-common/src/main/java/com/azure/android/communication/common/PhoneNumberIdentifier.java

```diff
@@ -30,6 +30,35 @@ public String getPhoneNumber() {
         return phoneNumber;
     }
 
+    /**
+     * Checks if the phone number is anonymous, e.g., used to represent a hidden caller ID.
+     *
+     * @return true if the phone number is anonymous, false otherwise.
+     */
+    public boolean isAnonymous() {
+        return getRawId().equals(PHONE_NUMBER_PREFIX + ANONYMOUS);
```

Comment:
maybe we can extend with private field and set in the constructor, instead of checking the equality everytime, wdyt?

Reply:
Set in constructor.

Reply:
So I realized I do not know if IsAnonymous reflect rawId which can be set later. Or constructor parameters.